Test/Test-WhiskeyPowerShellModule.Tests.ps1

```diff
@@ -0,0 +1,152 @@
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+$globalModules = @()
+$output = $null
+
+function Init
+{
+    $script:globalModules = @()
+    $script:output = $null
+}
+
+function GivenModuleInstalled
+{
+    param(
+        [Parameter(Mandatory)]
+        [String]$Name,
+
+        [Parameter(Mandatory)]
+        [String]$AtVersion
+    )
+
+    $script:globalModules += [pscustomobject]@{
+        'Name' = $Name;
+        'Version' = $AtVersion;
+    }
+
+    $script:globalModules | Should -Not -BeNullOrEmpty

```

Comment:
Remove this assertion, no need for it here. This function just sets up the test environment.